From 1cd4323b940408aa34dcaa01bd8a7ed43d9a966a Mon Sep 17 00:00:00 2001
From: Scott Moser <smoser@brickies.net>
Date: Thu, 1 Jun 2017 12:40:12 -0400
Subject: [PATCH] azure: remove accidental duplicate line in merge.

In previous commit I inadvertantly left two calls to
   asset_tag = util.read_dmi_data('chassis-asset-tag')
The second did not do anything useful.  Thus, remove it.
---
 cloudinit/sources/DataSourceAzure.py | 1 -
 1 file changed, 1 deletion(-)

--- a/cloudinit/sources/DataSourceAzure.py
+++ b/cloudinit/sources/DataSourceAzure.py
@@ -326,7 +326,6 @@ class DataSourceAzureNet(sources.DataSou
         if asset_tag != AZURE_CHASSIS_ASSET_TAG:
             LOG.debug("Non-Azure DMI asset tag '%s' discovered.", asset_tag)
             return False
-        asset_tag = util.read_dmi_data('chassis-asset-tag')
         ddir = self.ds_cfg['data_dir']
 
         candidates = [self.seed_dir]
